auditorium/src/views/components/auditorium/format.js

```diff
@@ -19,18 +20,21 @@ const Format = (props) => {
       break
     case 'count':
       value = formatCount(source)
+      tooltip = value !== Math.floor(source)
       break
     case 'percentage':
       value = formatNumber(source, 100, 1) + ' %'
       break
     default:
       value = formatNumber(source, factor, digits)
   }
-  return (
-    <Fragment>
-      {value}
-    </Fragment>
-  )
+  return tooltip
+    ? (
+      <span title={source}>
+        {value}
+      </span>
+    )
+    : value
```

Comment:
Today I learned you can just return a string from a (P)React component without having to wrap it in anything. Interesting!